GirLoader/Output/Model/Alias.cs

```diff
@@ -1,16 +1,16 @@
 ﻿using System.Collections.Generic;
 
-namespace Repository.Model
+namespace GirLoader.Output.Model
 {
     public class Alias : Element
     {
-        public Namespace Namespace { get; }
+        public Repository Repository { get; }
```

Comment:
I'm not a fan of using Repository and Namespace interchangeably like this. I think it's an unnecessary indirection, given that the Generator will only ever use the Namespace field. I would like to keep using Namespace here like before.